packages/fluentui/docs/src/components/Playground/renderConfig.ts

```diff
@@ -24,60 +24,74 @@ export const babelConfig = {
   presets: ['es2015'],
 };
 
-export const imports: Record<string, { version: string; module: any }> = {
+type Import = {
+  module: any;
+  version: string;
+  required: boolean;
+};
+
+export const imports: Record<string, Import> = {
   '@fluentui/accessibility': {
     version: accessibilityPackageJson.version,
     module: Accessibility,
+    required: false,
   },
   '@fluentui/code-sandbox': {
-    version: 'latest',
+    version: sandboxPackageJson.version,
     module: CodeSandbox,
+    required: true,
   },
   '@fluentui/docs-components': {
     version: docsComponentsPackageJson.version,
     module: DocsComponent,
+    required: true,
   },
   '@fluentui/react-icons-northstar': {
     version: projectPackageJson.version,
     module: FluentUIIcons,
+    required: false,
   },
   '@fluentui/react-northstar': {
     version: projectPackageJson.version,
     module: FluentUI,
+    required: true,
   },
   '@fluentui/react-bindings': {
     version: projectPackageJson.dependencies['@fluentui/react-bindings'],
     module: Bindings,
+    required: false,
   },
   classnames: {
     version: projectPackageJson.dependencies['classnames'],
     module: Classnames,
+    required: false,
   },
   lodash: {
     version: projectPackageJson.dependencies['lodash'],
     module: _,
+    required: false,
   },
   react: {
     version: projectPackageJson.peerDependencies['react'],
     module: React,
+    required: true,
   },
   'react-dom': {
     version: projectPackageJson.peerDependencies['react-dom'],
     module: ReactDOM,
-  },
-  'react-fela': {
```

Comment:
Would avoid removing dependencies with these changes (although we think it is not used)

Reply:
Agree, reverted